plugins/modules/npm.py

```diff
@@ -191,6 +203,7 @@ def __init__(self, module, **kwargs):
                 registry=cmd_runner_fmt.as_opt_val('--registry'),
                 no_optional=cmd_runner_fmt.as_bool('--no-optional'),
                 no_bin_links=cmd_runner_fmt.as_bool('--no-bin-links'),
+                force=cmd_runner_fmt.as_bool('--force'),
```

Comment:
You also need to add `force` to the command line (that first arg, str, with the name of all the params used in CLI), otherwise the runner will not add it to the CLI call. 

See https://docs.ansible.com/ansible/latest/collections/community/general/docsite/guide_cmdrunner.html for more details.

Reply:
oops, thanks for the doc!